rpc-rs/tests/nats_sub.rs

```diff
@@ -1,95 +1,102 @@
 //! test nats subscriptions (queue and non-queue) with rpc_client
 #![cfg(test)]
 
-const THREE_SEC: Duration = Duration::from_secs(3);
-
-use std::{str::FromStr as _, time::Duration};
+use std::{str::FromStr as _, sync::Arc, time::Duration};
```

Comment:
Super nit: this `str::FromStr as _` is very non-standard in my experience, just importing `FromStr` is fine

Reply:
I've gotten in the habit of using underscore that for traits that need to be in scope but aren't referenced directly., partly because some other traits like `Write` have more common names.  FromStr is pretty standard though, and the underscore makes it slightly harder to read. Removed.